cloudhsmtool/Dockerfile

```diff
@@ -1,38 +1,48 @@
 FROM ubuntu:18.04
-RUN apt-get update && apt-get install wget dnsutils -y
-RUN apt-get install multiarch-support libbsd0 -y
-RUN wget https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/Xenial/cloudhsm-client_latest_amd64.deb
-RUN wget https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/Xenial/cloudhsm-client-jce_latest_amd64.deb
-RUN wget https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/Xenial/cloudhsm-client-dyn_latest_amd64.deb
-RUN wget http://de.archive.ubuntu.com/ubuntu/pool/main/j/json-c/libjson-c2_0.11-4ubuntu2_amd64.deb
-RUN wget http://de.archive.ubuntu.com/ubuntu/pool/main/libe/libedit/libedit2_3.1-20150325-1ubuntu2_amd64.deb
-RUN dpkg -i libjson-c2_0.11-4ubuntu2_amd64.deb
-RUN dpkg -i libedit2_3.1-20150325-1ubuntu2_amd64.deb
-RUN apt-get install libjson-c2 libedit2 -y
-RUN dpkg -i cloudhsm-client_latest_amd64.deb
-RUN dpkg -i cloudhsm-client-jce_latest_amd64.deb
-RUN dpkg -i cloudhsm-client-dyn_latest_amd64.deb
-RUN apt-get install openjdk-11-jdk -y
-RUN apt-get install python -y
+
+ENV CLASSPATH=/opt/cloudhsm/java/*
+ENV HSM_PARTITION=PARTITION_1
+ENV LD_LIBRARY_PATH=/opt/cloudhsm/lib
+
+RUN apt-get update && apt-get install   wget \
+                                        dnsutils \
+                                        multiarch-support \
+                                        libbsd0 \
+                                        openjdk-11-jdk \
+                                        python \
+                                        rbenv \
+                                        ruby-build \
+                                        git \
+                                        autoconf \
+                                        bison \
+                                        -y
```

Comment:
previously you `libjson-c2` and `libedit2` were being installed via `apt` but nor are not? Are they needed?

Reply:
Ah, looks like they're being grabbed by `wget` and installed with `dpkg` now.